src/Tools/Test Framework/Test Runner/src/SelectTestRunner.Page.al

```diff
@@ -11,9 +11,10 @@ page 130454 "Select TestRunner"
 {
     Editable = false;
     PageType = List;
-    SourceTable = AllObjWithCaption;
-    SourceTableView = where("Object Type" = const(Codeunit),
-                            "Object Subtype" = const('TestRunner'));
+#pragma warning disable AS0035
+    SourceTable = "CodeUnit Metadata";
```

Comment:
Breaking change?

Reply:
Double checked Telemetry, there are no SaaS usage for those two pages